cleanse_answers.py

```diff
@@ -0,0 +1,47 @@
+import json
+import re
+
+def sanitize_text(text: str) -> str:
```

Comment:
can you sanitize while we save

Reply:
we sanitize when we get from page  
this approach is additional and used mostly for people who didn't upgrade previously